packages/assets-controllers/src/TokenDetectionController.ts

```diff
@@ -437,105 +477,100 @@ export class TokenDetectionController extends StaticIntervalPollingController<
     networkClientId?: NetworkClientId;
     accountAddress?: string;
```

Comment:
Since we now have a `#selectedAddress` property, what do you think about renaming this to `selectedAddress`? This would match the [extension version](https://github.com/MetaMask/metamask-extension/blob/e0704deda5967e9df3450d523a55b1a7f241d0ce/app/scripts/controllers/detect-tokens.js#L129), and would also match other places such as `addDetectedTokens` in TokensController.

Reply:
Applied here: https://github.com/MetaMask/core/pull/3923/commits/38b1ee0cb1ba479e17b6f11dfd5a8e15687ef9f7